roles/nighthawk/tasks/http-tasks.yml

```diff
@@ -0,0 +1,24 @@
+---
+# tasks for http termination type.
+
+- name: Create service for http type server pods
+  k8s:
+    definition: "{{ lookup('template', 'http-service.yml.j2') | from_yaml }}"
+  with_sequence: start=0 count={{ workload_args.number_of_routes | default(1)  | int }}
+  vars:
+    type: http
+  when: 
+    - workload_args.serviceip is defined and workload_args.serviceip
+    - ( workload_args.service_type | default("ClusterIP") == "NodePort" )
```

Comment:
I think you can remove `| default("ClusterIP") == "NodePort"` ?

Reply:
Done as we have decided to keep clusterIP as default